# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1334454308 25200
#      Sat Apr 14 18:45:08 2012 -0700
# Branch stable
# Node ID 05b7fa3b64c8a8b62c5fa08b491ff19c6f2cbb62
# Parent  6eabd81604b577c7ec8f27aeb6f4391d7f001ac6
Restore 'TextureMap' option for surface objects (Bug #35661)

* graphics.h.in (surface::properties): Add 'TextureMap' to option
to 'FaceColor' property.

diff --git a/src/graphics.h.in b/src/graphics.h.in
--- a/src/graphics.h.in
+++ b/src/graphics.h.in
@@ -4572,17 +4572,17 @@ public:
       array_property ydata u , Matrix ()
       array_property zdata u , Matrix ()
       array_property cdata u , Matrix ()
       radio_property cdatamapping al , "{scaled}|direct"
       string_property xdatasource , ""
       string_property ydatasource , ""
       string_property zdatasource , ""
       string_property cdatasource , ""
-      color_property facecolor , "{flat}|none|interp"
+      color_property facecolor , "{flat}|none|interp|texturemap"
       double_radio_property facealpha , double_radio_property (1.0, radio_values ("flat|interp"))
       color_property edgecolor , color_property (color_values (0, 0, 0), radio_values ("flat|none|interp"))
       radio_property linestyle , "{-}|--|:|-.|none"
       double_property linewidth , 0.5
       radio_property marker , "{none}|s|o|x|+|.|*|<|>|v|^|d|p|h|@"
       color_property markeredgecolor , "{auto}|none"
       color_property markerfacecolor , "auto|{none}"
       double_property markersize , 6
